### PR TITLE
feat: Auto reviewer branch policy

### PR DESCRIPTION
**Pull request by C123R**
Signed-off-by: Cizer Pereira <mail@cizer.dev>

## All Submissions:

* [X] Have you added an explanation of what your changes do and why you'd like us to include them?
* [X] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.
* [X] All new and existing tests passed.
* [X] My code follows the code style of this project.
* [X] I ran lint checks locally prior to submission.
* [X] Have you checked to ensure there aren't other open PRs for the same update/change?

## What about the current behavior has changed?

Issue Number: #29 

## Does this introduce a change to `go.mod`, `go.sum` or `vendor/`?

- [X] Yes
- [ ] No

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

## Any relevant logs, error output, etc?

(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -11,6 +11,9 @@ require (
 	github.com/hashicorp/terraform-plugin-sdk v1.10.0
 	github.com/microsoft/azure-devops-go-api/azuredevops v1.0.0-b3
 	github.com/stretchr/testify v1.4.0
-	golang.org/x/crypto v0.0.0-20200427165652-729f1e841bcc
+	golang.org/x/crypto v0.0.0-20200622213623-75b288015ac9
```
Comment (EliiseS) [inline go.mod]: Why the changes to the go.mod?

File: azuredevops/internal/service/policy/resource_branchpolicy_auto_reviewers.go [hunk 2]
```diff
@@ -0,0 +1,124 @@
+package policy
```
Comment (EliiseS) [inline azuredevops/internal/service/policy/resource_branchpolicy_auto_reviewers.go:1]: Could you add an acceptance test for this too? :)

Comment (C123R) [reply azuredevops/internal/service/policy/resource_branchpolicy_auto_reviewers.go:1]: Done!

File: go.mod (hunk 1 above)
Comment (C123R) [reply go.mod]: should be fine now.

File: website/docs/r/branch_policy_auto_reviewers.html.markdown [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+resource "azuredevops_git_repository" "r" {
+  project_id = azuredevops_project.p.id
+  name       = "Sample Repo"
+  initialization {
+    init_type = "Clean"
+  }
+}
+
```
Comment (EliiseS) [inline website/docs/r/branch_policy_auto_reviewers.html.markdown:26]: Can you format this terraform? Put it in a tf file, format it and paste it back? I can see a few things not being spaced correctly

Comment (C123R) [reply website/docs/r/branch_policy_auto_reviewers.html.markdown:26]: better now? terraform extension on vscode driving me crazy 😶

Comment (EliiseS) [reply website/docs/r/branch_policy_auto_reviewers.html.markdown:26]: Looks great! I'm sorry, I'm channeling @nmiodice with this level of nitpicking. 😆 (He's told me off so many times for my improper terraform in docs) 😄

File: go.mod (hunk 1 above)
Comment (C123R) [reply go.mod]: @EliiseS Good to merge?

Comment (EliiseS) [reply go.mod]: Yup! I was just running the integration tests :)

**Review by EliiseS — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (EliiseS) [reply go.mod]: @C123R Are you able to press the merge button? If not I'll do it :)

Comment (C123R) [reply go.mod]: Nope, I am missing write access!

**Comment by EliiseS** [comment]
@C123R Thank you!

**Comment by C123R** [comment]
> @C123R Thank you!

Awesome, @EliiseS Thanks for your support :)

Btw, there are other branch policies which we could add it as a resource. Should I create an issue first or directly come up with the PR?

**Comment by EliiseS** [comment]
@C123R An issue first, but more as a place of documentation, such as links to the API docs and design discussion 😄 
Feel free to start working on it right away, just make a comment on the issue saying you're picking it up.
